node/heartbeat/realTimer.go

```diff
@@ -0,0 +1,20 @@
+package heartbeat
+
+import "time"
+
+// RealTimer is an implementation of Timer and uses real time.now
+type RealTimer struct {
+}
+
+// Now returns the time.Now() Time
+func (m *RealTimer) Now() time.Time {
+	return time.Now()
+}
+
+// IsInterfaceNil() verifies if the interface is nil
```

Comment:
Delete the parentheses from the comment.

Reply:
fixed